src/views/Dashboard.vue

```diff
@@ -0,0 +1,136 @@
+<!--
+Nextcloud - Activity
+
+@author Jakob Röhrl
+@copyright 2021 Jakob Röhrl <jakob.roehrl@web.de>
+
+@author Raimund Schlüßler
+@copyright 2021 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+License as published by the Free Software Foundation; either
+version 3 of the License, or any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<template>
+	<DashboardWidget
+		id="activity_panel"
+		:items="activities"
+		:show-more-text="t('activity', 'activities')"
+		:show-more-url="showMoreUrl"
+		:loading="loading"
+		:half-empty-content-message="t('activity', 'No activities')">
+		<template #default="{ item }">
+			<!-- {{item._activity}} -->
+			<DashboardWidgetItem
+				:id="item.activity_id"
+				:target-url="item.link"
+				:avatar-username="item.user"
+				:overlay-icon-url="item.icon"
+				:main-text="item.dateFromNow"
+				:sub-text="item.subject" />
```

Comment:
```suggestion
				:main-text="item.subject"
				:sub-text="item.dateFromNow" />
```

The action is more important than the when.